### ISSUE TITLE
BlockControlStructureSpacing: Expected 1 line after "case", found 0

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1183]
> SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure
> Expected 1 line after "case", found 0

```php
    public function a() : void
    {
        switch (true) {
            case 'a':
                $b = true;

                break;
            default:
                $b = false;
        }
    }
```

I believe it should not be reported. Using default sniff settings.

The error is controlled by `BlockControlStructureSpacingSniff::$linesCountAfter`.

When set to 0, it disappears. However, that gets in conflict with desired behaviour:

> Expected 0 lines after "foreach", found 1.

**Comment by kukulich** [comment]
It's expected behaviour.

**Comment by simPod** [comment]
@kukulich I did not find a way to satisfy the sniff TBH not to be in conflict with `SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountAfterLastControlStructure`

![image](https://user-images.githubusercontent.com/327717/112454064-99afb600-8d58-11eb-9fc9-e120f928ab91.png)

> SlevomatCodingStandard.ControlStructures.JumpStatementsSpacing.IncorrectLinesCountAfterLastControlStructure:
> Expected 0 lines after "break", found 1.

```php
    public function a() : void
    {
        switch (true) {
            case 'a':
                $b = true;

                break;

            default:
                $b = false;
        }
    
```

**Comment by kukulich** [comment]
@simPod You have to modify the `controlStructures` settings.

**Comment by simPod** [comment]
Interesting. So I did override it to exclude `case`.

```xml
    <rule ref="SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing">
        <properties>
            <property name="controlStructures" type="array">
                <element value="default" />
                <element value="do" />
                <element value="if" />
                <element value="for" />
                <element value="foreach" />
                <element value="switch" />
                <element value="try" />
                <element value="while" />
            </property>
        </properties>
    </rule>
```

Thanks.
